packages/docs/base/icons.md

```diff
@@ -0,0 +1,206 @@
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Close
+        </td>
+        <td>
+          To close a modal or other UI
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Completed
+        </td>
+        <td>
+          To show a completed process
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Forward
+        </td>
+        <td>
+          To navigate forward
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Backward
+        </td>
+        <td>
+          To navigate backward
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Edit
+        </td>
+        <td>
+          To edit something
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Settings
+        </td>
+        <td>
+          To edit user or app settings
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Add
+        </td>
+        <td>
+          To add
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Subtract
+        </td>
+        <td>
+          To subtract
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Download
+        </td>
+        <td>
+          To indicate a user can download something
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</figure>
+
+## Built on a grid
+
+Every icon is built on a grid at a specific size. This ensures not only visual consistency, but allows us to easily swap icons in various applications.
+
+### Design your own
+
+This grid is also included as a component in Figma. If you need to create something new, just drag out an instance and start drawing.
+
+## Guidelines
+
+### UI indicators vs. icons
+
+UI indicators are visual affordances that are baked into UIs like the radio circle, checkbox check, and select caret. In some cases we can override these indicators to match our overall aesthetics and style.
+
+### When to use an icon
+
+If you want to call attention to an action you want a user to take, icons make good targets and break up text.
+
+### Use words
+
+Almost all icons should be accompanied by copy. By doing so, you won't leave users guessing what an icon represents.
+
+As best practice, use a verb to suggest an action. For more clarity use a verb + noun.
+
+### Choosing the right icon
+
+It’s recommended to use a word to accompany an icon. Like the previous example shows, icons should call attention to the action and the icon should follow suit. Depending on the situation, some icons are better for clicking, where others are better suited as static.
+
+As a result, this will drastically reduce the amount of icons we have, increase understanding and clarity in our actions.
+
+### Page structure first
```

Comment:
@joejezowski-okta @arnoldsandoval-okta @johnintrater-okta should we move this up the page since it ends with "first"?